<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="silius" errors="0" failures="65" skipped="13" tests="482" time="254.679" timestamp="2025-04-16T11:37:44.080086+00:00" hostname="fv-az1701-735"><testcase classname="" name="tests.single.gas.test_pre_verification_gas_calculation" time="0.000"><skipped message="collection skipped">('/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/gas/test_pre_verification_gas_calculation.py', 15, 'Skipped: Slow and requires fixing')</skipped></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_priority_fee_bump]" time="0.246" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_max_fee_bump]" time="0.168" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_same_fee]" time="0.193" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_less_fee]" time="0.165" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_below_threshold]" time="0.168" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_at_threshold]" time="0.274" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_above_threshold]" time="0.271" /><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_unstaked_sender" time="0.790" /><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_staked_sender" time="0.945" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_op_access_other_ops_sender_in_bundle" time="0.811" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_sender_double_role_in_bundle" time="0.421" /><testcase classname="tests.single.bundle.test_bundle" name="test_stake_check_in_bundler" time="0.547" /><testcase classname="tests.single.bundle.test_codehash" name="test_codehash_changed" time="0.243"><skipped type="pytest.skip" message="no self destruct. can't check code change..">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_codehash.py:54: no self destruct. can't check code change..</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_paymaster" name="test_paymaster_deposit" time="0.702"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x4e1303fa57024d9d6b4f0e03dcf27af7110e86d28fb7f8ef8d8303b42b35ec6b', id=98)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_paymaster.py:48: in test_paymaster_deposit
    assert_rpc_error(res, "too low", RPCErrorCode.PAYMASTER_DEPOSIT_TOO_LOW)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x4e1303fa57024d9d6b4f0e03dcf27af7110e86d28fb7f8ef8d8303b42b35ec6b', id=98)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-]" time="0.382" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-CALL:&gt;]" time="0.605" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.701" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-]" time="0.398" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-CALL:&gt;]" time="0.627" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-DELEGATECALL:&gt;]" time="0.713" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-]" time="0.407" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-CALL:&gt;]" time="0.666" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-DELEGATECALL:&gt;]" time="0.722" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-]" time="0.531"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-CALL:&gt;]" time="0.785"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-DELEGATECALL:&gt;]" time="0.872"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-]" time="0.526"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-CALL:&gt;]" time="0.776"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-DELEGATECALL:&gt;]" time="0.865"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-]" time="0.548" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-CALL:&gt;]" time="0.819" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-DELEGATECALL:&gt;]" time="0.897" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-]" time="0.607" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-CALL:&gt;]" time="0.896" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.995" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-]" time="0.447" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-CALL:&gt;]" time="0.693" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.865" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-]" time="0.548" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-CALL:&gt;]" time="0.801" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-DELEGATECALL:&gt;]" time="0.905" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-]" time="0.564" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.799" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.887" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-]" time="0.600" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.978" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.958" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-]" time="1.153" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-CALL:&gt;]" time="1.027" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="1.125" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]unstaked][paymaster][context][ok-]" time="0.244"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='A unstaked paymaster at 0x9b37b715966f71f767d681c60aa57796d0f308d4: must not return context', data=None, id=172)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='A unstaked paymaster at 0x9b37b715966f71f767d681c60aa57796d0f308d4: must not return context', data=None, id=172)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]unstaked][paymaster][context][ok-CALL:&gt;]" time="0.270"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='A unstaked paymaster at 0x99edbe4dd0a0ed7a070a491ca2574a7c1ebb0c22: must not return context', data=None, id=174)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='A unstaked paymaster at 0x99edbe4dd0a0ed7a070a491ca2574a7c1ebb0c22: must not return context', data=None, id=174)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]unstaked][paymaster][context][ok-DELEGATECALL:&gt;]" time="0.306"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='A unstaked paymaster at 0xb6b2cf1baf7cb2050440d63f5fbb8d93bc678748: must not return context', data=None, id=176)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='A unstaked paymaster at 0xb6b2cf1baf7cb2050440d63f5fbb8d93bc678748: must not return context', data=None, id=176)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-]" time="0.596" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-CALL:&gt;]" time="0.742" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.967" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-]" time="0.599" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.708" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.906" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-]" time="0.474" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.708" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.797" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-]" time="0.394" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-CALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.801" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-]" time="0.525" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-CALL:&gt;]" time="0.658" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-DELEGATECALL:&gt;]" time="0.728" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-]" time="0.424" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-CALL:&gt;]" time="0.656" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-DELEGATECALL:&gt;]" time="0.842" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-]" time="0.571" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-CALL:&gt;]" time="0.828" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-DELEGATECALL:&gt;]" time="0.921" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.632" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="0.908" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.141" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.733" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="0.919" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.989" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-]" time="0.487" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-CALL:&gt;]" time="0.693" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.978" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-]" time="0.560" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.815" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="1.005" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-]" time="0.620" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.894" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.971" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-]" time="0.779" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.261" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.242" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]staked][paymaster][context][ok-]" time="0.276" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]staked][paymaster][context][ok-CALL:&gt;]" time="0.393" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]staked][paymaster][context][ok-DELEGATECALL:&gt;]" time="0.334" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-]" time="0.638" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-CALL:&gt;]" time="0.882" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.975" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-]" time="0.621" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-CALL:&gt;]" time="0.881" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.957" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-]" time="0.644" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.833" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.919" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-]" time="0.494" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.841" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.921" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-]" time="0.421" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-CALL:&gt;]" time="0.494" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-]" time="0.431" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-CALL:&gt;]" time="0.611" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-DELEGATECALL:&gt;]" time="0.623" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-]" time="0.429" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-CALL:&gt;]" time="0.609" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-DELEGATECALL:&gt;]" time="0.633" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-]" time="0.350" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-CALL:&gt;]" time="0.664" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-DELEGATECALL:&gt;]" time="0.715" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-]" time="0.357" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-CALL:&gt;]" time="0.709" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.768" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-]" time="0.332" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-CALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.528" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-]" time="0.464" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-CALL:&gt;]" time="0.653" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-DELEGATECALL:&gt;]" time="0.714" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-]" time="0.342" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-CALL:&gt;]" time="0.598" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-]" time="0.345" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-CALL:&gt;]" time="0.647" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.587" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-]" time="0.368" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-CALL:&gt;]" time="0.639" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-DELEGATECALL:&gt;]" time="0.537" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-]" time="0.309" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.683" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.725" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-]" time="0.409" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.559" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.726" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-]" time="0.310" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.558" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.619" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-]" time="0.304" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.613" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.560" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-]" time="0.403" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.533" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.592" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-]" time="0.310" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.668" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.627" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-]" time="0.412" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.551" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.613" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-]" time="0.544" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-CALL:&gt;]" time="0.718" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.626" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-]" time="0.364" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.547" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.562" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-]" time="0.450" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-CALL:&gt;]" time="0.518" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.537" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-]" time="0.356" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-CALL:&gt;]" time="0.625" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-DELEGATECALL:&gt;]" time="0.546" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-]" time="0.358" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-CALL:&gt;]" time="0.639" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-DELEGATECALL:&gt;]" time="0.652" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-]" time="0.489" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-CALL:&gt;]" time="0.613" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-DELEGATECALL:&gt;]" time="0.754" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-]" time="0.489" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-CALL:&gt;]" time="0.637" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.695" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-]" time="0.359" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-CALL:&gt;]" time="0.551" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.566" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-]" time="0.463" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-CALL:&gt;]" time="0.569" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.637" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-]" time="0.371" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-CALL:&gt;]" time="0.718" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.890" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-]" time="0.478" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-CALL:&gt;]" time="0.576" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.628" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-]" time="0.373" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-CALL:&gt;]" time="0.671" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.722" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-]" time="0.562" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-CALL:&gt;]" time="0.739" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.673" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-]" time="0.491" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.676" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.594" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-]" time="0.669" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-CALL:&gt;]" time="0.581" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.570" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-]" time="0.400" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-CALL:&gt;]" time="0.605" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.596" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-]" time="0.407" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-CALL:&gt;]" time="0.530" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.629" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-]" time="0.411" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-CALL:&gt;]" time="0.463" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="0.410" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-]" time="0.943"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-CALL:&gt;]" time="0.787"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
CALL:&gt; funding test_rules_target 0x5C25f932a81cA6fc277f07B8b4f90620cbDf383f

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="0.689"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-]" time="1.001"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.198"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.272"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-]" time="0.346" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.658" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-]" time="0.421" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-CALL:&gt;]" time="0.539" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.633" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-]" time="0.409" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-CALL:&gt;]" time="0.703" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.693" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-]" time="0.334" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.539" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.636" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-]" time="0.311" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-CALL:&gt;]" time="0.605" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.601" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-]" time="0.337" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-CALL:&gt;]" time="0.535" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.627" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-]" time="0.329" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-CALL:&gt;]" time="0.665" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.662" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-]" time="0.334" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.575" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.674" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-]" time="0.432" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-CALL:&gt;]" time="0.722" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.747" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-]" time="0.456" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.563" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.555" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-]" time="0.439" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-CALL:&gt;]" time="0.558" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.564" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-]" time="0.444" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-CALL:&gt;]" time="0.572" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.645" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-]" time="0.435" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-CALL:&gt;]" time="0.663" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-DELEGATECALL:&gt;]" time="0.666" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-]" time="0.426" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-CALL:&gt;]" time="0.554" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-DELEGATECALL:&gt;]" time="0.546" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-]" time="0.444" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-CALL:&gt;]" time="0.680" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-DELEGATECALL:&gt;]" time="0.673" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-]" time="0.451" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-CALL:&gt;]" time="0.715" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-DELEGATECALL:&gt;]" time="0.704" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-]" time="0.373" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.555" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.517" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-]" time="0.382" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.520" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.616" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-]" time="0.380" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.520" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.616" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-]" time="0.288" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-CALL:&gt;]" time="0.608" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-DELEGATECALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-]" time="0.287" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-CALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-DELEGATECALL:&gt;]" time="0.601" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-]" time="0.390" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-CALL:&gt;]" time="0.509" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-DELEGATECALL:&gt;]" time="0.510" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-]" time="0.370" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.694" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.640" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-]" time="0.370" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.504" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.608" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-]" time="0.370" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.521" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.512" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-]" time="0.268" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.653" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.507" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-]" time="0.290" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-CALL:&gt;]" time="0.653" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-DELEGATECALL:&gt;]" time="0.658" /><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx" time="0.078"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0xe06c…c8e3 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:36: in test_send_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0xe06c…c8e3 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_initcode" time="0.075"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:85: in test_send_eip_7702_tx_with_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_flag_no_initcode" time="0.075"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:127: in test_send_eip_7702_tx_with_flag_no_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[0]" time="0.075"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0xe5e2…f150 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0xe5e2…f150 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[1337]" time="0.076"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x6f5b…dd8e is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x6f5b…dd8e is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_bad_eip_7702_drop_userop" time="0.077"><failure message="AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32602 = Error(code=-32602, message='sender 0x8dba…1a3a is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code">tests/single/eip7702/test_eip7702_tuple_userop.py:208: in test_send_bad_eip_7702_drop_userop
    assert_rpc_error(response, "", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32602 = Error(code=-32602, message='sender 0x8dba…1a3a is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_nonsender_eip_7702_drop_userop" time="0.165"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x59d77a06c366660e85b389113d234d83937931f122f68db585ddbd7316a6ea51', id=682)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:223: in test_send_nonsender_eip_7702_drop_userop
    assert_rpc_error(userop.send(), "sender", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x59d77a06c366660e85b389113d234d83937931f122f68db585ddbd7316a6ea51', id=682)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_wrongchain_eip_7702_drop_userop" time="0.072"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x0e41…6ea3 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:244: in test_send_wrongchain_eip_7702_drop_userop
    assert_ok(userop.send())
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x0e41…6ea3 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE]" time="0.239" /><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE2]" time="0.341" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE]" time="0.403" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE2]" time="0.318" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create2_once" time="0.396" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE,expect=ok,rule=OP-032]" time="0.293"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=696)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=696)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE2,expect=err,rule=OP-032]" time="0.298"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f456e970930&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f456e970930&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e973930&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f456e973930&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=698).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f456e970930&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f456e970930&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e973930&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f456e973930&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=698).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE,expect=ok,rule=EREP-060]" time="0.422"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=700)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=700)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE,expect=err,rule=EREP-060]" time="0.420"><failure message="AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e973990&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f456e973990&gt; = 'account uses banned opcode: CREATE'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e971d10&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f456e971d10&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=702).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e973990&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f456e973990&gt; = 'account uses banned opcode: CREATE'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e971d10&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f456e971d10&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=702).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE2,expect=err,rule=EREP-060]" time="0.419"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f456e9728b0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f456e9728b0&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e970a50&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f456e970a50&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f456e9728b0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f456e9728b0&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f456e970a50&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f456e970a50&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE2,expect=ok,rule=EREP-060]" time="0.318"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-]" time="0.215" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-CALL:&gt;]" time="0.468" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.477" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-]" time="0.212" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-CALL:&gt;]" time="0.368" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.493" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-]" time="0.213" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.490" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.399" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-]" time="0.318" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-CALL:&gt;]" time="0.382" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.501" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-]" time="0.218" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.397" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.404" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-]" time="0.223" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-CALL:&gt;]" time="0.484" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.429" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-]" time="0.324" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.488" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.503" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-]" time="0.226" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-CALL:&gt;]" time="0.491" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.409" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-]" time="0.349" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.525" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.435" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-]" time="0.334" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-CALL:&gt;]" time="0.499" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.512" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-]" time="0.328" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-CALL:&gt;]" time="0.485" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.406" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-]" time="0.332" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.502" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.540" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-]" time="0.343" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-CALL:&gt;]" time="0.400" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.518" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-]" time="0.241" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-CALL:&gt;]" time="0.513" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.523" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-]" time="0.374" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.541" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.439" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-]" time="0.241"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xd45f4ad38f5077e8e458ff83d9ad683fbd1da51f4d3a7e7abd7235940baa8ded', id=798)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xd45f4ad38f5077e8e458ff83d9ad683fbd1da51f4d3a7e7abd7235940baa8ded', id=798)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.411"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xf2835008bb25cc8c145b81c2f8aeb65ed904fce7b04349a104379a272e8f1c82', id=800)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xf2835008bb25cc8c145b81c2f8aeb65ed904fce7b04349a104379a272e8f1c82', id=800)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.529"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x92488f7fe6eb5d4e8759cd957ee2f0a080c6c4342e6ae62212f9268d6c5bfe42', id=802)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x92488f7fe6eb5d4e8759cd957ee2f0a080c6c4342e6ae62212f9268d6c5bfe42', id=802)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-]" time="0.348"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x5aeb2b6fa5ef1820f4c060e59c495aaf3090fe35f2fba485bb3bbb7580c5e757', id=804)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x5aeb2b6fa5ef1820f4c060e59c495aaf3090fe35f2fba485bb3bbb7580c5e757', id=804)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.418"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x748044b0a1a8f0247b7b4c3a057e7234a43da4f9992da105761d645f821dc589', id=806)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x748044b0a1a8f0247b7b4c3a057e7234a43da4f9992da105761d645f821dc589', id=806)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.440"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xee0e95c057c624f8deef49c9ab56b9dd450be82f41a3255fdf44bfdf411d98aa', id=808)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xee0e95c057c624f8deef49c9ab56b9dd450be82f41a3255fdf44bfdf411d98aa', id=808)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-]" time="0.256"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=810)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=810)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-CALL:&gt;]" time="0.517"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=812)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=812)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-DELEGATECALL:&gt;]" time="0.541"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=814)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=814)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-]" time="0.273"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=816)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=816)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-CALL:&gt;]" time="0.547"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=818)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=818)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-DELEGATECALL:&gt;]" time="0.471"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=820)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=820)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-]" time="0.375" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-CALL:&gt;]" time="0.541" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-DELEGATECALL:&gt;]" time="0.460" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-]" time="0.378" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-CALL:&gt;]" time="0.481" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-DELEGATECALL:&gt;]" time="0.605" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-]" time="0.374" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-CALL:&gt;]" time="0.549" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.648" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-]" time="0.377" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-CALL:&gt;]" time="0.581" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.678" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-]" time="0.408" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.638" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.725" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-]" time="0.457" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-CALL:&gt;]" time="0.629" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.711" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-]" time="0.423" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.544" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.629" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-]" time="0.394" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-CALL:&gt;]" time="0.612" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.698" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-]" time="0.408" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.634" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.629" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-]" time="0.419" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-CALL:&gt;]" time="0.624" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.723" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-]" time="0.461" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.688" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.763" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-]" time="0.403" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-CALL:&gt;]" time="0.609" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.711" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-]" time="0.290" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-CALL:&gt;]" time="0.621" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.591" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-]" time="0.421" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.690" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.734" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-]" time="0.405" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-CALL:&gt;]" time="0.612" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.740" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-]" time="0.425" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-CALL:&gt;]" time="0.643" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.712" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-]" time="0.462" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.686" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.773" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-]" time="0.423"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xe60dce9d498bc07771ac86e2d434ae1cc0596aaa34bc04ac83eafdada9f90298', id=924)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xe60dce9d498bc07771ac86e2d434ae1cc0596aaa34bc04ac83eafdada9f90298', id=924)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.635"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x995f710c57f07701ac1b4358162facfe6224e73ca9a7bd753eb9333ff60ce33e', id=926)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x995f710c57f07701ac1b4358162facfe6224e73ca9a7bd753eb9333ff60ce33e', id=926)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.738"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x6feb00fa0647de8606503edab9027109b9dbc193673d9dcae653cd97389b4d2d', id=928)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x6feb00fa0647de8606503edab9027109b9dbc193673d9dcae653cd97389b4d2d', id=928)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-]" time="0.450"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xe2b49edcfedd7a0ae9552f43bf5083140846738ebd9dc3591c7eebe1e025e6f6', id=930)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xe2b49edcfedd7a0ae9552f43bf5083140846738ebd9dc3591c7eebe1e025e6f6', id=930)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.676"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x031ad88a37bd23072e070b9efd581d0a0b5fafe631b901e1bec11e4f7cea1ebb', id=932)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x031ad88a37bd23072e070b9efd581d0a0b5fafe631b901e1bec11e4f7cea1ebb', id=932)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.758"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x92a5b892cb577ccd008424726bd8f58425c3e239b41519f34ab219832a1529bf', id=934)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x92a5b892cb577ccd008424726bd8f58425c3e239b41519f34ab219832a1529bf', id=934)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[BALANCE]" time="0.431"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=936)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=936)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[SELFBALANCE]" time="0.474"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=938)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=938)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GAS]" time="0.367" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[NUMBER]" time="0.370" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[TIMESTAMP]" time="0.270" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[COINBASE]" time="0.373" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[DIFFICULTY]" time="0.274" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BASEFEE]" time="0.374" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASLIMIT]" time="0.375" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASPRICE]" time="0.381" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFBALANCE]" time="0.400" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BALANCE]" time="0.381" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[ORIGIN]" time="0.389" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOCKHASH]" time="0.283" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE]" time="0.394" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE2]" time="0.400" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFDESTRUCT]" time="0.396" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBHASH]" time="0.394"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x1ceff72f84eeb46db50ff2d9005c626a78136f221832bad3f9570aa1bed65e42', id=970)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x1ceff72f84eeb46db50ff2d9005c626a78136f221832bad3f9570aa1bed65e42', id=970)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBBASEFEE]" time="0.396"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x51979f0196890f4040f4e6ac0e4818f312bbd4b89bd12e3391a8ca4cf9a5b10f', id=972)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x51979f0196890f4040f4e6ac0e4818f312bbd4b89bd12e3391a8ca4cf9a5b10f', id=972)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[BALANCE]" time="0.404"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=974)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=974)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[SELFBALANCE]" time="0.428"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=976)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=976)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_paymaster_on_account_failure" time="0.293"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/reputation/test_erep.py:71: in test_paymaster_on_account_failure
    set_reputation(paymaster.address, ops_seen=5, ops_included=2)
tests/utils.py:321: in set_reputation
    assert res.result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_staked_factory_on_account_failure" time="0.860"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String(&quot;0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000&quot;)))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=986)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/reputation/test_erep.py:119: in test_staked_factory_on_account_failure
    send_bundle_now()
tests/utils.py:230: in send_bundle_now
    assert_ok(RPCRequest(method="debug_bundler_sendBundleNow").send(url))
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String("0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000")))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=986)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[staked]" time="0.491"><failure message="AssertionError: staked account should be blamed instead of paymaster&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 1 diff: Reputation(address='0x81a5afd3cf94783ffd0333e59c96b274ee991949', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x81a5afd3cf94783ffd0333e59c96b274ee991949', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0x7a751792fb5d39088080db59892943535ce2ad13',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0x81a5afd3cf94783ffd0333e59c96b274ee991949',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:174: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: staked account should be blamed instead of paymaster
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 1 diff: Reputation(address='0x81a5afd3cf94783ffd0333e59c96b274ee991949', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x81a5afd3cf94783ffd0333e59c96b274ee991949', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0x7a751792fb5d39088080db59892943535ce2ad13',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0x81a5afd3cf94783ffd0333e59c96b274ee991949',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
staking account &lt;web3._utils.datatypes.Contract object at 0x7f456e85fb20&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[unstaked]" time="0.389"><failure message="AssertionError: valid userop. staked paymaster should have opsSeen increment&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 0 diff: Reputation(address='0x9473b8d1d298f12b25cb9b84ba160a17b09a8fce', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x9473b8d1d298f12b25cb9b84ba160a17b09a8fce', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0x9473b8d1d298f12b25cb9b84ba160a17b09a8fce',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0xe753c8b648f18d9696078bbf1f45c457842111df',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:156: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: valid userop. staked paymaster should have opsSeen increment
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 0 diff: Reputation(address='0x9473b8d1d298f12b25cb9b84ba160a17b09a8fce', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x9473b8d1d298f12b25cb9b84ba160a17b09a8fce', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0x9473b8d1d298f12b25cb9b84ba160a17b09a8fce',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0xe753c8b648f18d9696078bbf1f45c457842111df',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
unstaked account &lt;web3._utils.datatypes.Contract object at 0x7f456ede38b0&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_replace" name="test_replace_paymaster" time="0.579"><failure message="AssertionError: replace with unmodified userop&#10;assert 2 == 1&#10; +  where 2 = pm_opsSeen('0xc41120a0d91E4F1b18033FE1cFF4C6e7D9E2a950')&#10; +    where '0xc41120a0d91E4F1b18033FE1cFF4C6e7D9E2a950' = &lt;web3._utils.datatypes.Contract object at 0x7f456eda3820&gt;.address">tests/single/reputation/test_replace.py:37: in test_replace_paymaster
    assert pm_opsSeen(pm1.address) == 1, "replace with unmodified userop"
E   AssertionError: replace with unmodified userop
E   assert 2 == 1
E    +  where 2 = pm_opsSeen('0xc41120a0d91E4F1b18033FE1cFF4C6e7D9E2a950')
E    +    where '0xc41120a0d91E4F1b18033FE1cFF4C6e7D9E2a950' = &lt;web3._utils.datatypes.Contract object at 0x7f456eda3820&gt;.address</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[with_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[without_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_chainId" name="test_eth_chainId[]" time="0.009" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas[]" time="0.179" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_execution_revert" time="0.185"><failure message="AttributeError: expected error object, got:&#10;Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1014)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/rpc/test_eth_estimateUserOperationGas.py:32: in test_eth_estimateUserOperationGas_execution_revert
    assert_rpc_error(response, "", RPCErrorCode.EXECUTION_REVERTED)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1014)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_simulation_revert" time="0.031"><failure message="assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: &quot;AA23 reverted&quot;,...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1016).code">tests/single/rpc/test_eth_estimateUserOperationGas.py:40: in test_eth_estimateUserOperationGas_simulation_revert
    assert_rpc_error(response, "dead signature", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: "AA23 reverted",...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1016).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash[]" time="0.367"><failure message="jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas&#10;&#10;Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:&#10;    {'title': 'paymaster verification gas limit',&#10;     'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},&#10;               {'type': 'number'}],&#10;     'description': 'The amount of gas to allocate for the paymaster '&#10;                    'validation code'}&#10;&#10;On instance['userOperation']['paymasterVerificationGasLimit']:&#10;    None">tests/single/rpc/test_eth_getUserOperationByHash.py:24: in test_eth_getUserOperationByHash
    validate(instance=response.result, schema=schema)
.venv/lib/python3.10/site-packages/jsonschema/validators.py:1332: in validate
    raise error
E   jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas
E   
E   Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:
E       {'title': 'paymaster verification gas limit',
E        'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},
E                  {'type': 'number'}],
E        'description': 'The amount of gas to allocate for the paymaster '
E                       'validation code'}
E   
E   On instance['userOperation']['paymasterVerificationGasLimit']:
E       None</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash_error" time="0.005" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt[]" time="0.380" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt_error" time="0.005" /><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation[]" time="0.348"><failure message="AssertionError: assert '0x3ba20c567a...ad4ae2b7d6b9a' == '0xc0e352eb17...a52b57b69a14a'&#10;  &#10;  - 0xc0e352eb1724971353db5556c1f4f6d4baba2fa39de891aa09da52b57b69a14a&#10;  + 0x3ba20c567af791330723aca78eb80e246e9020c6a63ceb68f7dad4ae2b7d6b9a">tests/single/rpc/test_eth_sendUserOperation.py:19: in test_eth_sendUserOperation
    assert response.result == userop_hash(helper_contract, userop)
E   AssertionError: assert '0x3ba20c567a...ad4ae2b7d6b9a' == '0xc0e352eb17...a52b57b69a14a'
E     
E     - 0xc0e352eb1724971353db5556c1f4f6d4baba2fa39de891aa09da52b57b69a14a
E     + 0x3ba20c567af791330723aca78eb80e246e9020c6a63ceb68f7dad4ae2b7d6b9a</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_revert" time="0.043"><failure message="AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1033).code">tests/single/rpc/test_eth_sendUserOperation.py:32: in test_eth_sendUserOperation_revert
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1033).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_invalid_signature" time="0.030"><failure message="AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;&#10; +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1036).code">tests/single/rpc/test_eth_sendUserOperation.py:39: in test_eth_sendUserOperation_invalid_signature
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;
E    +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1036).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_supportedEntryPoints" name="test_eth_supportedEntryPoints[]" time="0.010" /></testsuite></testsuites>